index.html

```diff
@@ -1412,6 +1412,91 @@ <h3>Biometrics</h3>
+      document.
+    </p>
+
+    <section class="informative">
+      <h3>Broad Ethical Principles</h3>
+      <p>
+        Decentralized Identifiers may come to underpin much of our digital life. This 
+        may include our public social and career personas, as well our private personas 
+        shared between friends and family. Identifiers representing these personas and 
+        relationships are some of the most important in people's lives, and great care 
+        should be taken when securing an identifier system that supports these critical 
+        activities. As with all things, strong consideration for the appropriate and ethical 
+        use of technology should be made when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">Principles</a>
+        section of the Ethical Web Principles, there are certain key goals that should 
+        apply to all Web standards and technologies. DIDs explicitly support several
+        of these goals, especially the following:
+        <ul>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">Privacy</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">Individual Control</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">Device Independence</a></li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Environmental Considerations</h3>
+      <p class="advisement">
+        The following section reflects the views of some members of the working group.
+        Additional PRs are welcome from the working group with additional points of view.
+      </p>
+
+      <section class="informative">
+        <h3>Environmental Principles as Applied to DID method development</h3>
+
+        <p>
+          The guiding principle that 
+          <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
+          web must be an environmentally sustainable platform</a> should be followed.
+          <br />
+          When implementing or utilizing a DID method, consideration should be given
+          to the environmental impacts of any underlying technologies. 
+        </p>
+
+        <p>
+          Utilizing or authoring DID methods that compromise on
+          the principles of security and / or control for other principles
+          such as environmental concerns should be very carefully 
+          considered in balance against alternative approaches that may
+          provide higher levels of security and protection.
+          The specific use case for a DID method will help determine the
+          optimal approach.  For matters that concern human rights, selection 
+          of approaches that are weighted towards privacy and control should 
+          outweigh other considerations.
```

Comment:
```suggestion
          Utilizing or authoring DID methods that trade principles such as security
          or control for other principles such as environmental concerns should be 
          very carefully considered against alternative approaches that balance 
          these principles differently. The specific use case for a DID method will
          help determine the optimal approach.  In matters that concern human
          rights, approaches that are weighted towards privacy and control will 
          typically outweigh other considerations.
```

Reply:
I feel like this really softens the security, privacy, human rights vs environment stance, basically making all of them seem like equivalent concerns. I do not believe these are equivalent concerns, as the former are matters of immediate personal safety and individual harms, which may even include defense against violence against human beings. For these reason I would like to see the original text maintained.

Reply:
They *are* equivalent concerns, in the abstract! The user of the rubric is the only person who can judge which axis of concern (if any) takes a primary, seconday, etc., position, which judgement they must make based on the context of their deployment/implementation/usage-scenario — which may be, but is probably not, exactly the same as yours.

Reply:
> They *are* equivalent concerns, in the abstract!

This is a good argument for folding the "Environmental Considerations" section into a simpler EWP section that mentions none of them in particular, but forwards the entire section on to the rubric.

Reply:
Exactly - if the argument is they are equivalent concerns (I still personally put human rights first), then it doesn't make sense to have a whole one-off section about it. We should remove this one-off callout section and push all considerations to the rubric.

Reply:
Agreed. This entire section should go.